docs/development/docs.rst

```diff
@@ -10,27 +10,84 @@ The docs are kept in the ``docs/`` directory at the top of the source tree.
    However, if there is going to be periodic/bigger contributions, it may be a
    good idea to install the Sphinx requirements to build our docs.
 
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation,
+you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+#. click the checkmark next to your commit and it will expand to have multiple options
+#. right-click the "details" link next to the "docs/readthedocs.org:docs" item
```

Comment:
I think an image could be included below this list

![Screenshot_2021-04-14 Improve documentation on contributing to documentation by cocobennett · Pull Request #8082 · readthed](https://user-images.githubusercontent.com/4975310/114787277-268cd480-9d45-11eb-9c2f-cb5066d34045.png)